portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,20 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  const quotes =
+      ['I\'m ready!', 'The best time to wear a striped sweater is all the time.',
```

Comment:
put each item on a separate line so they are easier to read

Reply:
Fixed